asv/commands/quickstart.py

```diff
@@ -51,7 +48,7 @@ def run(cls, dest=".", top_level=None):
             color_print("(2) benchmark suite in a separate repository")
             color_print("")
             while True:
-                answer = raw_input("Layout to use? [1/2] ")
+                answer = input("Layout to use? [1/2] ")
```

Comment:
When I change this function from python 2 raw_input() to input() I get this error. What am I doing wrong?

```
    def test_quickstart(tmpdir):
        tmpdir = str(tmpdir)
    
        dest = join(tmpdir, 'separate')
        os.makedirs(dest)
    
        tools.run_asv('quickstart', '--no-top-level', '--dest', dest)
    
        assert isfile(join(dest, 'asv.conf.json'))
        assert isfile(join(dest, 'benchmarks', 'benchmarks.py'))
        conf = util.load_json(join(dest, 'asv.conf.json'), js_comments=True)
        assert 'env_dir' not in conf
        assert 'html_dir' not in conf
        assert 'results_dir' not in conf
    
        dest = join(tmpdir, 'same')
        os.makedirs(dest)
    
        try:
            asv.commands.quickstart.raw_input = lambda msg: '1'
>           tools.run_asv('quickstart', '--dest', dest)

test/test_quickstart.py:30: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
test/tools.py:148: in run_asv
    return args.func(args, **kwargs)
asv/commands/quickstart.py:40: in run_from_args
    return cls.run(dest=args.dest, top_level=args.top_level)
asv/commands/quickstart.py:54: in run
    answer = input("Layout to use? [1/2] ")
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <_pytest.capture.DontReadFromInput object at 0x7f622daa2eb0>, args = ()

    def read(self, *args):
>       raise OSError(
            "pytest: reading from stdin while output is captured!  Consider using `-s`."
        )
E       OSError: pytest: reading from stdin while output is captured!  Consider using `-s`.

../../../.local/lib/python3.8/site-packages/_pytest/capture.py:192: OSError
--------------------------------------------- Captured stdout call ----------------------------------------------

· Setting up new Airspeed Velocity benchmark suite.
· Edit asv.conf.json to get started.
· Setting up new Airspeed Velocity benchmark suite.

Which of the following template layouts to use:
(1) benchmark suite at the top level of the project repository
(2) benchmark suite in a separate repository

Layout to use? [1/2] 
============================================ short test summary info ============================================
FAILED test/test_quickstart.py::test_quickstart - OSError: pytest: reading from stdin while output is captured...
=============================================== 1 failed in 1.01s ===============================================

```

Reply:
Not sure, but same as for the other, do you mind splitting this to its own PR please?